lib/exportJob.js

```diff
@@ -350,9 +378,9 @@ class ExportJob extends EventEmitter {
       }
     } else {
       pageDim = pageDimensions[pageSize]
+      // Flip if landscape
+      pageDim = landscape ? {x: pageDim.y, y: pageDim.x} : pageDim
```

Comment:
I updated the test instead, it was technically a bug so I'd rather let the test document the expected behavior.  Thanks for asking.